specification/paths/Addresses.json

```
@@ -0,0 +1,181 @@
+{
+  "get": {
+    "tags": [
+      "Addresses"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "organizations.manage",
+          "shipments.manage"
+        ]
+      }
+    ],
+    "summary": "Get all addresses for a user.",
+    "description": "This endpoint retrieves all address entries that the requesting user has access to.",
+    "parameters": [
+      {
+        "name": "filter[search]",
+        "in": "query",
+        "description": "String of characters you want to look for in an address.",
+        "schema": {
+          "type": "string"
+        }
+      },
+      {
+        "name": "filter[organization]",
+        "in": "query",
+        "description": "Comma separated string of organization ids to filter by.",
+        "schema": {
+          "type": "string"
+        }
```

This would be nice (also on other places), but we should rename that file because it's not shipment specific.
```suggestion
        "$ref": "#/components/parameters/query-shipments-filter-organization"
```